client/web/src/repo/blob/BlameColumn.tsx

```diff
@@ -128,7 +128,7 @@ export const BlameColumn = React.memo<BlameColumnProps>(({ isBlameVisible, codeV
                         onSelect={selectRow}
                         onDeselect={deselectRow}
                     />,
-                    portalRoot.querySelector(`.${styles.wrapper}`) as HTMLDivElement
+                    portalRoot.querySelector(`.${styles.wrapper.replaceAll('+', '\\+')}`) as HTMLDivElement
```

Comment:
Maybe use `getElementsByClassName` instead? This should also be faster in theory since we don't have to parse the CSS selector. 🙈 

```suggestion
                    portalRoot.getElementsByClassName(styles.wrapper)[0] as HTMLDivElement
```

<img width="1175" alt="Screenshot 2022-12-07 at 15 23 17" src="https://user-images.githubusercontent.com/458591/206204006-802bdb4c-6023-4a1c-9229-2551046f367a.png">

Reply:
ah sure, that feels better :)

Reply:
done, PTAL!